packages/lake/src/components/LakeAlert.tsx

```diff
@@ -131,7 +136,7 @@ export const LakeAlert = ({
       </Box>
 
       {isNotNullish(children) && (
-        <View style={icon != null ? styles.content : null}>
+        <View style={icon != null && desktop ? styles.contentDesktop : styles.contentMobile}>
```

Comment:
Instead of creation a new style to cancel the margin, you can only apply it when needed:

```tsx
<View style={desktop && icon != null && styles. content}>
```